### DISCUSSION TITLE
Lab 5 Buttons

### DISCUSSION THREAD

**Discussion by joempappas** [ag-informatics/ag-informatics-course#28] [Troubleshooting]
Hello,

What constitutes a "major action"? Would navigating to a different page be considered a major action? For instance if we have a nav bar at the top of a page where the buttons in a button group will help with site navigation.

**Comment by StevenDoylePurdue** [comment]
Yes, that would be a major action. As long as you have at least 2 buttons that "do something," you should be good.
